### ISSUE TITLE
How to get rid of top header that is 100px tall

### ISSUE THREAD

**Issue by cheverdyukv** [cloudscribe/cloudscribe.SimpleContent#438]
I found that style brand-header has 100px height and it is not possible to remove it with any option. 
`.brand-header {
    height: 100px;
}`
and there is following div
`<div class="fixed-top container-fluid brand-header bg-white">`

Is it intentional?

**Comment by joeaudette** [comment]
Themes are not part of SimpleContent, they are things added to your app by the project template. First make sure you are using the latest project template. There have been many iterations on themes in the project template. In cloudscribe core we introduced an idea for header and footer content and site logo image a while ago but it was difficult to make that work with included themes without making some things consistent across them, otherwise adding a logo or header or footer content threw the layout out of wack. Even now it can if you put too much header or footer content. Ultimately if you want to change something about a theme it lives in your project and you own it. Some versions of project template we set hard coded styles from theme config settings, sounds like you have one of those versions of project template. Look in the layout file for style attributes set by theme config. Update to latest template and shouldn’t be as much of that.

**Comment by cheverdyukv** [comment]
It is how it come after executing 
`dotnet new cloudscribe -D NoDb -S a -L true`

check file `sitefiles\s1\themes\custom1\wwwroot\css\style.css` and you will find this:
`.brand-header {
  height: 100px; }`
But I actually checked in browser and it is the same

**Comment by joeaudette** [comment]
Seems you are not understanding, please reinstall the project template to get latest version and create new project to get latest themes. Or look in your theme layout file for style attributes and remove or change them as you wish. Once an app is generated you own the themes and must maintain them I cannot change them after the fact. Any issues in the themes we ship can only be fixed by shipping s new project template, no way for us to fix existing projects.

**Comment by cheverdyukv** [comment]
I'm executing these two lines:
`dotnet new -i "cloudscribe.templates::*"`
`dotnet new cloudscribe -D NoDb -S a -L true`

Is there anything else I should execute to get latest templates?

**Comment by joeaudette** [comment]
Ok if you have latest template then that style comes from either style attribute in layout or from css of the theme both of which are in your control not mine after project is created.

**Comment by cheverdyukv** [comment]
Sorry perhaps I wasn't clear. After creating new project from latest templates by default there is 100px header. I didn't do any modifications to it, just created new project and ran it. After checking source code I find no way to remove it with settings. Of course I can edit .css file and remove it but before that I'm asking you if this is only way to remove that header. I would assume that this is big change to add header and I though there should be way to remove it without editing files.

I'm trying to keep my version latest and periodically I created new project and bring all changes from it to my version. The less changes I have to remember about the better.

If there is no other ways I will edit .css file.

**Comment by joeaudette** [comment]
Yes you have to edit the css. As I said before we came up with some standards that worked reasonably well for header and footer editable content in cloudscribe core. We made the header 100px to accommodate a site logo and limited header content. It works reasonably but not perfectly for themes we ship in the template. But in all cases you now own those versions of the themes in your app so if you want changes yes of corse you have to edit css. And you need to regen minified css for production.

**Comment by joeaudette** [comment]
If you add a site logo or header content in site settings I think you will see why we did that. Still it is not a perfect solution for every theme and if you add more than one line of text as header content the layout can still go out of wack so css tweaks may be needed to make header content look good and not throw if other aspects of layout.

**Comment by cheverdyukv** [comment]
Just out of curiosity, why did you decide to used fixed height?
